src/main/java/com/jadc/bazaar/controller/AccountController.java

```diff
@@ -0,0 +1,73 @@
+package com.jadc.bazaar.controller;
+
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.bson.Document;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.jadc.bazaar.entity.Account;
+import com.jadc.bazaar.service.AccountService;
+import com.jadc.bazaar.service.CustomerService;
+import com.mongodb.BasicDBObject;
+import com.mongodb.MongoCredential;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoDatabase;
+@RestController
+@RequestMapping("/customer")
+public class AccountController {
+
+	private final AccountService accountService;
+
+	@Value("${admin.bzrdb.username}")
+	private  String ADMIN_USERNAME;
+
+	@Value("${admin.bzrdb.password}")
+	String  ADMIN_PASSWORD;
+
+	@Value("${admin.bzrdb.url}")
+	String  ADMIN_URL;
+
+	@Autowired
+	public AccountController(AccountService service) {
+		this.accountService = service;
+	}
+
+
+	@PutMapping("/create_db/{id}")
+	public void createDatabase(@PathVariable Integer id){
+		String adminRole = "readWrite";
+		String userRole = "read";
+
+		Account account = accountService.createCredentials(id);
```

Comment:
I was thinking of using `MongoDBCustomerAccount` rather than `MongoDBAccount` because this will also lead to confusion since there's a MongoDB login online.